src/Microsoft.Health.Dicom.Blob/Features/Storage/BlobFileStore.cs

```diff
@@ -58,12 +60,24 @@ public class BlobFileStore : IFileStore
         var blobUploadOptions = new BlobUploadOptions { TransferOptions = _options.Upload };
         stream.Seek(0, SeekOrigin.Begin);
 
+        BlobContentInfo info = null;
+
         await ExecuteAsync(async () =>
```

Comment:
I would if we should have a variant of this `ExecuteAsync` that returns the `T` in `Task<T>`?